autopush_rs/src/util/ddb_helpers.rs

```diff
@@ -182,6 +184,17 @@ impl From<NotificationHeaders> for HashMap<String, String> {
     }
 }
 
+impl From<HashMap<String, String>> for NotificationHeaders {
+    fn from(val: HashMap<String, String>) -> NotificationHeaders {
+        NotificationHeaders {
+            crypto_key: val.get("crypto_key").map(|v| v.to_string()),
+            encryption: val.get("encrypption").map(|v| v.to_string()),
```

Comment:
"encryption"

Reply:
Also, not sure what the impact of setting these to the string "None" might be. Might be better to set them to an empty string?

Reply:
They won't be included if they're None, as the serde tag says to skip serializing them in that case.